### PR TITLE
Pref(dark-mode): Add Card Popup Dark mode adaptation, restyle part of Card and Message

### PR DESCRIPTION
**Pull request by yzh990918**

### CODE REVIEW

**Comment by priority3** [comment]
Maybe I can add `dark mode` button in playground😶

**Comment by zyyv** [comment]
> Maybe I can add `dark mode` button in playground😶

Maybe we can try the effect first.
In the future, I want to refacotor the playground. Now the playground more means preview, the demos should displayed on docs.

File: packages/preset/src/shortcuts/card.ts [hunk 1]
```diff
@@ -1,18 +1,23 @@
...
   'o-card-header': 'fb px-4 py-2.5',
   'o-card-header-wrapper': 'flex flex-col flex-1 gap-1.5',
-  'o-card-header-title': 'text-md',
-  'o-card-header-description': 'text-xs c-context:60 break-all !m-0',
+  'o-card-header-title': 'text-md dark:c-#ECECEC!',
+  'o-card-header-description':
+    'text-xs c-context:60 dark:c-rgba(255,255,255,.7) break-all !m-0 dark:c-white!',
   'o-card-header-extra': '',
-  'o-card-body': 'p-4 pt-1.5 leading-normal text-sm',
+  'o-card-body': 'p-4 pt-1.5 leading-normal text-sm dark:c-#7C7C7D!',
```
Comment (zyyv) [inline packages/preset/src/shortcuts/card.ts]: Or we can put the color as a suitable name into the theme.

File: packages/preset/src/shortcuts/card.ts [hunk 2]
```diff
@@ -1,18 +1,23 @@
 export const cardShortcuts: Record<string, string> = {
   // card
-  'o-card-bg': 'bg-gradient-linear bg-gradient-from-rgba(255, 255, 255, 0.2) bg-gradient-to-rgba(255, 255, 255, 0.035) backdrop-blur-lg',
-  'o-card-base': 'o-card-bg o-transition list-none relative rounded-md c-context w-full of-hidden',
+  'o-card-bg':
+    'bg-gradient-linear bg-gradient-from-rgba(255, 255, 255, 0.2) bg-gradient-to-rgba(255, 255, 255, 0.035) backdrop-blur-lg',
+  'o-card-base':
+    'o-card-bg o-transition list-none relative rounded-md c-context w-full of-hidden',
```
Comment (zyyv) [inline packages/preset/src/shortcuts/card.ts]: I like to align other lines without a new line.

**Review by zyyv — CHANGES_REQUESTED** [review]

**Comment by priority3** [comment]
> > Maybe I can add `dark mode` button in playground😶
> 
> Maybe we can try the effect first. In the future, I want to refacotor the playground. Now the playground more means preview, the demos should displayed on docs.

yeah, 😊

File: packages/preset/src/shortcuts/card.ts (hunk 1 above)
Comment (yzh990918) [reply packages/preset/src/shortcuts/card.ts]: done

File: packages/preset/src/shortcuts/card.ts (hunk 2 above)
Comment (yzh990918) [reply packages/preset/src/shortcuts/card.ts]: Caused by vscode auto format 😲.

**Review by zyyv — APPROVED** [review]

**Pull request merged**